app/components/gh-theme-error-li.js

```diff
@@ -2,5 +2,18 @@ import Component from 'ember-component';
 
 export default Component.extend({
     tagName: '',
-    error: null
+    error: null,
+    showDetails: false,
+
+    actions: {
+        toggleDetails() {
+            let toggleDetails = this.get('showDetails');
```

Comment:
Ember has a built in method for toggling properties, you could do `this.toggleProperty('showDetails')`

Alternatively, we pull in the `ember-composable-helpers` addon in Ghost-Admin so it would be possible to do this in the template directly:

```hbs
<a href="" class="theme-validation-toggle-details" {{action (toggle "showDetails")}} data-test-toggle-details>
```